modules/extensions/src/terrain/terrain-picking-pass.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Layer,
+  Viewport,
+  LayersPassRenderOptions,
+  _PickLayersPass as PickLayersPass
+} from '@deck.gl/core';
+import type {default as TerrainCover} from './terrain-cover';
+
+export type TerrainPickingPassRenderOptions = LayersPassRenderOptions & {
+  pickZ: boolean;
+};
+
+/** Renders textures used by the TerrainEffect picking pass */
+export default class TerrainCoverPass extends PickLayersPass {
```

Comment:
Two classes with the same name. Dropping default exports would have sorted this immediately.